gnome-extra/krb5-auth-dialog/krb5-auth-dialog-43.0.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson gnome2-utils
+
+DESCRIPTION="An application to acquire and list Kerberos tickets."
+HOMEPAGE="https://gitlab.gnome.org/GNOME/krb5-auth-dialog/"
+SRC_URI="https://download.gnome.org/sources/${PN}/$(ver_cut 1)/${P}.tar.xz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	>=gui-libs/libadwaita-1.2.0:1
+	dev-libs/glib:2
+	x11-libs/gtk+:3
```

Comment:
`qa-vdb` also found `app-crypt/gcr:=` and `sys-libs/pam` as required RDEPEND

Reply:
right, that looks like https://packages.debian.org/bookworm/krb5-auth-dialog

the list i proposed came from `meson.build` from the sources, which might be incomplete